content/en/llm_observability/instrumentation/sdk.md

```diff
@@ -1245,11 +1245,15 @@ The `LLMObs.annotate()` method accepts the following arguments:
 
 `input_data`
 : optional - _JSON serializable type or list of dictionaries_
-<br />Either a JSON serializable type (for non-LLM spans) or a list of dictionaries with this format: `{"role": "...", "content": "..."}` (for LLM spans).  **Note**: Embedding spans are a special case and require a string or a dictionary (or a list of dictionaries) with this format: `{"text": "..."}`.
+<br />Either a JSON serializable type (for non-LLM spans) or a list of dictionaries with this format: `{"role": "...", "content": "..."}` (for LLM spans). Messages can also include optional `"tool_calls"` (list of dictionaries with required keys: `"name": "..."`, `"arguments": {...}` and optional keys: `"tool_id": "..."`, `"type": "..."`) and `"tool_results"` (list of dictionaries with required key: `"result": "..."` and optional keys: `"name": "..."`, `"tool_id": "..."`, `"type": "..."`) for function calling scenarios. **Note**: Embedding spans are a special case and require a string or a dictionary (or a list of dictionaries) with this format: `{"text": "..."}`.
```

Comment:
Maybe we reflect the docstrings attached to `LLMObs.annotate()` instead: https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/llmobs/_llmobs.py#L1320-L1325